src/monika/FibSeries.java

```diff
@@ -0,0 +1,27 @@
+
+package monika;
+
+/*Assignment - 37 :
+Print Fibonacci series for first 10 numbers.
+output : 0, 1, 1, 2, 3, 5, 8, 13, 21, 34
+*/
+
+
+public class FibSeries {
+
+	static void printFibonacciSeries(int length) {
+		int num1 = 0;
+		int num2 = 1;
+		System.out.print(num1 + ", " + num2 + ", ");
+		for (int i = 3; i <= length; i++) {
```

Comment:
why you started I's value from 3,
loop is independent it is just for iteration.

Reply:
you can start it from 1 or 0 as well, bcoz you are doing sum of currentNumber & currentNumber+1,